### ISSUE TITLE
funclatency to accept multiple events

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bcc#1108]
eg, to support:

```
# funclatency vfs_read vfs_write
```

In that case, we can do:

```
# funclatency 'vfs_(read|write)'
```

But other cases get trickier.

I'm also wondering if the -F logic should be flipped. So that if you match multiple functions, then by default split them in the output, and only use -F to merge them if desired.

**Comment by goldshtn** [comment]
Why are the other cases trickier? A single regular expression can cover an arbitrary set of strings, they don't have to overlap: `(vfs_write|__kmalloc|xfs_sync)`. What am I missing?

**Comment by ahrens** [comment]
Agreed that it's possible to write a regexp to match each of the functions that you want, e.g. `'^func1$|^func2$'`.  But this is a worse user experience than typing `func1 func2`. It would be a really nice usability improvement to accept multiple functions on the CLI.  After all, the point of this tool is a convenience to make it faster and more intuitive than doing the equivalent with bpftrace or BCC.

The example rexexps in the above 2 comments are unfortunately not accurate, because once you introduce metacharacters, it implicitly matches a subset, so e.g. `'zil_commit'` matches 1 function, but `'zil_commit|'` matches 10 functions.  I would take this as evidence that writing the appropriate regexp is *not* straightforward, and that justifies raising the priority of this issue.

**Comment by ahrens** [comment]
The same issue applies to `funccount`.
